### DISCUSSION TITLE
How can I make Prettier maintain a short multiline list?

### DISCUSSION THREAD

**Discussion by AutonomousCat** [prettier/prettier#16069] [Q&A]
Take this for example

```js
/** @type {import("prettier").Config} */
export default {
    plugins: ["prettier-plugin-astro", "prettier-plugin-tailwindcss"],
};
```

With Black formatter for Python, I would simply add a trailing comma after the last item and it would turn it into a multi-line list upon formatting.

```js
/** @type {import("prettier").Config} */
export default {
    plugins: [
        "prettier-plugin-astro",
        "prettier-plugin-tailwindcss",
    ],
};
```

Possibly a suggestion to make? Or is it already a setting I'm missing?

**Comment by kachkaev** [comment]
If having two lines in this code is critical for you, you can add `//` after `[` and it’ll do the trick:

**Prettier 3.2.5**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEB6AVOgBAARgTwAc4tgBLAW0IgCcYAKAHREJrhhjLhuYEoA6AMLQAZmQDmAXyzpUjKHAAe1OlgAmcEQEMArgBsYpeVhNZCeneLJQAzkiwBtLKjlRT7rM1btO3ALTmltZ+WjYwNBDMADTGHiZebBxcNAEWVlB+MFpkegDu1mpgNjbRsSYAujFQkgDcIFEgEISc0DbIoFo0EbkACp0IbShaeVr4bQ0ARjRaYADW7ADKWhRwADLWcMjaejZwk9Nzi4Qz1uLI4Tp7ILsUZOc0lw1KxDSUCFl6ACrcUJ1cg9tdg0bKc9HAAIo6CDwLbDIEgABWNkUC1BEKhMKQgKuAEcMXAehFCIMQKE-Ao4Bo1PUQOFsnpTsIKBQtMhSXo9DSQVBxGCAIIcV4THTwHrcdYKWE7K4ACxgFD0AHUZWR4DZjmA4AsBqqyAA3VX4NlFcYgPWXACSUA0sAWYFezT51oWBDBUvhrAgu0V00IbO8uxoes2DWsgZghK04hZ7quxxogbZEy0EzgnIarGsMEVZDUMBlyAAHAAGBpsPFkNiR6OsrFwq5ZCY5vMFpAAJgaOl2nxTAPrTwoqbUVNWWh5OijcAAYrQWUkeWzdDAICBJJIgA)

```sh
--parser babel
```

**Input:**

```jsx
/** @type {import("prettier").Config} */
export default {
    plugins: [ //
        "prettier-plugin-astro",
        "prettier-plugin-tailwindcss",
    ],
};
```

**Output:**

```jsx
/** @type {import("prettier").Config} */
export default {
  plugins: [
    //
    "prettier-plugin-astro",
    "prettier-plugin-tailwindcss",
  ],
};

```
